content/develop/interact/search-and-query/basic-constructs/configuration-parameters.md

```diff
@@ -212,6 +213,22 @@ $ redis-server --loadmodule ./redisearch.so MINPREFIX 3
 
 ---
 
+### MINSTEMLEN
+
+The minimum word length to stem. The default value is `4`. Setting it lower than `4` can hurt performance.
```

Comment:
```suggestion
The minimum word length to stem. The default value is `4`. Setting it lower than `4` can reduce performance.
```

Reply:
Thank you, @mich-elle-luna ! I'll push the change from my system.